### ISSUE TITLE
Build out header component

### ISSUE THREAD

**Issue by IanVS** [sem-js/sem-js#3]
The header should house main site navigation, which for now will just be the home page, speakers page, events page, and sponsors page.

**Comment by lucasvocos** [comment]
I can take this

**Comment by lucasvocos** [comment]
Pull request: https://github.com/sem-js/sem-js/pull/15

**Comment by wommy** [comment]
we need to build the navigation too?
- home
- speakers
- events
- sponsors

**Comment by lucasvocos** [comment]
Yes, that's taken by someone here: https://github.com/sem-js/sem-js/issues/12

Once that Navigation is built out, the Header should properly import the nav
